src/PowerShellEditorServices/Services/TextDocument/SemanticToken.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models.Proposals;
+
+namespace Microsoft.PowerShell.EditorServices.Services.TextDocument
+{
+    internal class SemanticToken
+    {
+        public SemanticToken(string text, SemanticTokenType type, int line, int column, IEnumerable<string> tokenModifiers)
+        {
+            Line = line;
+            Text = text;
+            Column = column;
+            Type = type;
+            TokenModifiers = tokenModifiers;
+        }
```

Comment:
Non-blocking but just wanted to say that if you get rid of this ctor, you can do:

```C#
new SemanticToken
{
    Text = "asdf",
    Line = 1,
    Column = 2,
   ...
}
```
and that will work the same way.

We can defer that for now.